### PR TITLE
add ros 1 github actions

### PR DESCRIPTION
**Pull request by flynneva**
- runs on PR to devel and master branches only
- runs on PR to ros1 and ros1/* branches only (no branches like this exist yet, but could in the future)

### CODE REVIEW

File: .github/workflows/ros_ci.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+name: ROS 1 CI
+
+on:
+  pull_request:
+    branches:
+      - 'develop'
+      - 'master'
```
Comment (mvollrath) [inline .github/workflows/ros_ci.yml]: master branch isn't maintained anymore and should be omitted.

**Review by mvollrath — CHANGES_REQUESTED** [review]

File: .github/workflows/ros_ci.yml (hunk 1 above)
Comment (flynneva) [reply .github/workflows/ros_ci.yml]:
```suggestion
```

**Comment by flynneva** [comment]
are there any ROS1 specific branches you'd like to run this on? we can add them to the triggers now

**Comment by mvollrath** [comment]
develop is good for now.  Whenever it's time to reconcile porting across the branches we can name it proper.

**Review by mvollrath — APPROVED** [review]

**Pull request merged**